system/web/context/ExceptionBean.cfc

```diff
@@ -35,7 +35,7 @@ component accessors="true"{
 		any extraMessage = "",
 		any extraInfo = ""
 	){
-		variables.exceptionStruct = arguments.errorStruct;
+		variables.exceptionStruct = duplicate(arguments.errorStruct);
```

Comment:
Brad,

without duplicate in CF-11 Type,Message, Tag Context, StackTrace are empty